lib/scrabble.rb

```diff
@@ -1,7 +1,36 @@
 class Scrabble
 
+  def standardized_format(word)
```

Comment:
I like that you broke out this method! Just make sure you're testing for any methods you create.